website/tests/mainPage.spec.ts

```diff
@@ -37,6 +41,8 @@ test.describe('Main page', () => {
                     .locator('..')
                     .getByText(linkName, { exact: true })
                     .click();
+                await expect(page.locator('text=Error -')).not.toBeVisible();
+                await expect(page.locator('text=Something went wrong')).not.toBeVisible();
```

Comment:
Doesn't `getByText` work here instead of `locator(...)`?

Reply:
My linter (prettier) complained and suggested this - it should do the same thing and getByTest makes some type assumptions that are apparently not accounted for in the tests